### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by hadobe**

### CODE REVIEW

**Review by skumar09 — APPROVED** [review]

File: configs/express.config.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ const config = {
         baseURL: envs['@express_live'],
       },
     },
+
+    {
+      name: 'express-live-webkit',
+      use: {
+        ...devices['Desktop Safari'],
+        baseURL: envs['@express_live'],
```
Comment (JackySun9) [inline configs/express.config.js:66]: if you run this on express live with automation, might cause hight traffic, do you have stage server

Comment (hadobe) [reply configs/express.config.js:66]: Is it true for Firefox and Chrome as well?

Comment (JackySun9) [reply configs/express.config.js:66]: yes, here you are using express_live, which is pointing to express hlx live, right?

Comment (hadobe) [reply configs/express.config.js:66]: yes, example - https://main--express--adobecom.hlx.live/drafts/nala/blocks/cards-dark

So you suggest I point to stage?

Comment (JackySun9) [reply configs/express.config.js:66]: Please check with your team to get suggestion?

Comment (JackySun9) [reply configs/express.config.js:66]: I just approve this now

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**